### PR TITLE
docs: generic schema for experimentation opportunities

### PR DESCRIPTION
**Pull request by ekremney**
Open API schema for the generic experimentation opportunities.

Example:

Screenshot 2024-08-12 at 19 21 14

### CODE REVIEW

**Review by ramboz — APPROVED** [review]

File: docs/openapi/schemas.yaml [hunk 1]
```diff
@@ -415,14 +415,58 @@ Audit:
...
+    page:
+      type: string
+      description: The title of the webpage
+      example: 'Retirement Calculator'
+    screenshot:
+      type: string
+      format: url
+      description: A URL pointing to the screenshot of the webpage.
+      example: "https://space.cat/screenshots/some-cool-page.png"
+    trackedPageConversion:
```
Comment (rpapani) [inline docs/openapi/schemas.yaml:451]: is there a list of types we allow or where do we get this information from?

File: docs/openapi/schemas.yaml [hunk 2]
```diff
@@ -415,14 +415,58 @@ Audit:
...
+          type: array
+          description: A list of experimentation opportunities
+          items:
+            oneOf:
+              - $ref: './schemas.yaml#/ExpOpportunityGeneric'
 AuditList:
   type: array
   items:
     $ref: './schemas.yaml#/Audit'
+ExpOpportunityGeneric:
```
Comment (rpapani) [inline docs/openapi/schemas.yaml:434]: opportunity can be with out experimentation as well right? like broken back links, translation etc. can we make the opportunity generic?

**Comment by rpapani** [comment]
@ekremney where can we specify the details/data/metrics of the opportunity so that the marketer can get a bit more understanding of the opportunity. For example, 
rage clicks opportunity - we may need to show information like, where the user is clicking, contents of the element where the rage click is happening, how many clicks etc.
decline conversion opportunity - we may need to show information like, how the conversion on a day to day is dropping so that the marketer can co relate the incidents and start stitching more data to accurately understand what went wrong and fix it.

**Review by rpapani — APPROVED** [review]

**Comment by ekremney** [comment]
>where can we specify the details/data/metrics of the opportunity so that the marketer can get a bit more understanding of the opportunity. For example,
rage clicks opportunity - we may need to show information like, where the user is clicking, contents of the element where the rage click is happening, how many clicks etc.
decline conversion opportunity - we may need to show information like, how the conversion on a day to day is dropping so that the marketer can co relate the incidents and start stitching more data to accurately understand what went wrong and fix it.

comment for the future, this topic was discussed in a meeting. cc/ @rpapani

**Pull request merged**